rodan-main/code/rodan/jobs/heuristic_pitch_finding/PitchFinding.py

```diff
@@ -131,19 +122,57 @@ def _x_projection_vector(self, glyph):
 
         if g.ncols > self.discard_size and g.nrows > self.discard_size:
             if g.ncols < self.avg_punctum:
-                this_punctum_size = g.ncols
+                this_punctum_size_cols = g.ncols
             else:
-                this_punctum_size = self.avg_punctum
+                this_punctum_size_cols = self.avg_punctum * 0.8
```

Comment:
Where does this 0.8 come from? If it's a heuristic, it should be defined elsewhere in the file and documented in comments (what it does, why it's at 0.8, acceptable range)